### ISSUE TITLE
`no-standalone-expect` doesn't seem to honour the `globalAliases` setting

### ISSUE THREAD

**Issue by sazzer** [playwright-community/eslint-plugin-playwright#357]
I'm creating a custom expectation for the first time, and I'm getting lint errors from the `no-standalone-expect` rule.

My expectation is this:
```
const expect = baseExpect.extend({
  async toBeAuthenticated(page: Page) {
    const assertionName = "toBeAuthenticated";
    let pass: boolean;
    let matcherResult: any;

    try {
      const homePage = await withPage(page);

      const navbar = homePage.navbar;
      await baseExpect(navbar.loginButton).toBeHidden();
      await baseExpect(navbar.userMenuButton).toBeVisible();

      pass = true;
    } catch (error: any) {
      matcherResult = error.matcherResult;
      pass = false;
    }

    const message = () => this.utils.matcherHint(assertionName, undefined, undefined, { isNot: this.isNot });

    return {
      message,
      pass,
      name: assertionName,
      actual: matcherResult?.actual
    };
  },
});
```

Which is copied almost exactly from [the documentation](https://playwright.dev/docs/test-assertions#add-custom-matchers-using-expectextend).

The `baseExpect` calls are both triggering the `no-standalone-expect` rule. However, I saw in issue #319 the suggestion to configure global aliases in the ESLint config.

So I've done that:
```
  {
    files: ["**/*.ts"],
    ignores: ["target/**", "node_modules/**"],
    plugins: {
      playwright,
    },
    rules: {
      ...playwright.configs["flat/recommended"].rules,

      "no-console": "error",
    },
    settings: {
      playwright: {
        globalAliases: {
          test: ["baseTest"],
          expect: ["baseExpect"]
        }
      }
    }
  },
```

And it's made absolutely no difference.

Is this an issue? Or am I configuring something wrong here?

Cheers

**Comment by carlbray** [comment]
I think it's an issue and not a problem with your config.

I wrote the following test which fails on the use of baseExpect in the new assertion.

```typescript
import rule from './no-standalone-expect.js'
import { runTSRuleTester, typescript } from '../utils/rule-tester.js'

runTSRuleTester('no-standalone-expect-ts', rule, {
  invalid: [
  ],
  valid: [
    // Global aliases
    {
      code: typescript`
        import { expect as baseExpect } from '@playwright/test';
        const expect = baseExpect.extend({
          async toBeAuthenticated() {
            const message = () => 'msg';
            baseExpect(1).toBe(1);
            return {
              message,
              pass: true,
              name: "toBeAuthenticated",
              actual: {}
            };
          },
        });
        test.describe('scenario', async () => {
          test('testing', () => await expect(1).toBeAuthenticated());
        });
      `,
      settings: { 
        playwright: {
          globalAliases: {
            expect: ['baseExpect'],
            test: ['it'],
          },
        },
      },
    },
  ],
})
```
